_oasis

```diff
@@ -10,6 +10,8 @@ Plugins:      DevFiles (0.3), META (0.3)
 
 Library stdext
   Path: lib
+  FindlibName: stdext
+  Pack: true
```

Comment:
I see you already did this in the next commit (didn't see from first glance at commit heading).